gtdynamics/utils/WalkCycle.cpp

```diff
@@ -44,8 +44,8 @@ std::map<string, Point3> WalkCycle::initContactPointGoal(
       // If no goal set yet, add it here
       if (cp_goals.count(link_name) == 0) {
         LinkSharedPtr link = robot.link(link_name);
-        auto foot_w = link->wTcom().transformFrom(kv.second.point) +
-                      robot.link("body")->wTcom().transformFrom(
+        auto foot_w = link->bTcom().transformFrom(kv.second.point) +
+                      robot.link("body")->bTcom().transformFrom(
```

Comment:
Use *, BUT is `robot.link(“body”)->bMcom()` not simply identity? I would add a unit test in every SDF read tests to assert this, and then this transformFrom simply disappears.

Reply:
relates to what we said in the earlier comment. I will add these asserts.

Reply:
removed the `robot.link("body")->bMcom`. I still need to add the asserts in the sdf and urdf reads in the tests.